fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1666,6 +1666,110 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'securityEvents',
+          function (t) {
+
+            function securityEvents(suite) {
```

Comment:
Probably worth noting that this file just got much bigger, any interest in putting securityEvents testing in its own file?